helm/aws-load-balancer-controller/templates/deployment.yaml

```diff
@@ -26,7 +26,7 @@ spec:
         {{- toYaml .Values.podLabels | nindent 8 }}
         {{- end }}
       annotations:
-        {{- if not .Values.serviceMonitor.enabled }}
+        {{- if .Values.prometheusScrapping.enabled }}
```

Comment:
You are effectively replacing the serviceMonitor.enabled with the new configuration. Did you meant either/or for those instead?

Reply:
Hi, 
I added `and` operator